CONTRIBUTING.md

```diff
@@ -5,6 +5,16 @@
 OpenSearch is a community project that is built and maintained by people just like **you**.
 [This document](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md) explains how you can contribute to this and related projects.
 
+### Design Documents
```

Comment:
We don't really discuss how to file bugs or feature requests in this doc - feels like a little much to include it

Reply:
Yeah sure